src/cli.ts

```diff
@@ -71,31 +73,7 @@ async function main() {
 
 	p.intro("🎉 Welcome to remult-cli!");
 
-	if (!args.connectionString) {
-		const answer = await p.group(
-			{
-				connectionString: async () =>
-					p.text({
-						message: `What's your connectionString?`,
-						placeholder:
-							"  (type it here or restart with --connectionString or a .env file containing DATABASE_URL)",
-						validate: (value) => {
-							if (value === "") {
-								return "Please enter something";
-							}
-
-							if (!value.startsWith("postgres")) {
-								return "Please enter a valid connexion string like: postgres://user:pass@host:port/db-name";
-							}
-						},
-					}),
-			},
-			{
-				onCancel: () => pCancel(),
-			},
-		);
-		args.connectionString = answer.connectionString;
-	}
+	args.connectionString ??= await getConnectionStringFromPrompt();
```

Comment:
Maybe I'm not used to `??=`!
But I find this less readable! :o

Reply:
I think you need to be used for it